toolchain/sem_ir/file.h

```diff
@@ -156,10 +156,12 @@ class File : public Printable<File> {
   auto facet_types() const -> const CanonicalValueStore<FacetTypeId>& {
     return facet_types_;
   }
-  auto complete_facet_types() -> ValueStore<CompleteFacetTypeId>& {
+  auto complete_facet_types()
+      -> RelationalValueStore<SemIR::FacetTypeId, SemIR::CompleteFacetTypeId>& {
```

Comment:
Perhaps it is worth creating a `typedef ... CompleteFacetTypeStore` so we don't have to repeat this long and tricky type name in so many places?

Reply:
Done (in the FIle class)